var/spack/repos/builtin/packages/ncurses/package.py

```diff
@@ -45,6 +45,7 @@ def flag_handler(self, name, flags):
 
     def configure(self, spec, prefix):
         opts = [
+            'ac_cv_path_LDCONFIG=true',
```

Comment:
Can you elaborate what this option does and why do we want it?

Reply:
The ncurses builds run `ldconfig` after install and then ignores the exit code, spack however still catches that error in the log. By overwriting `ldconfig` to be `true` is doesn't do that.

Using the `LDCONFIG` option would be possible too, but there are some OSes for which it gets overwritten again, hence using the configure internal option `ac_cv_path_LDCONFIG` seems better.